
<launch>
    <node pkg="pololu_controller" type="controller.py" name="controller" output="screen">
        <param name="arm_yaml" value="$(find pololu_controller)/launch/arm.yaml" />
        <param name="port_name" value="/dev/serial/by-id/usb-Pololu_Corporation_Pololu_Mini_Maestro_12-Channel_USB_Servo_Controller_00188316-if00" />
        <param name="baud_rate" value="115200" />
        <param name="rate_hz" value="10" />
        <param name="daisy_chain" value="false" />
    </node>
    <node pkg="pololu_controller" type="xbox_pub.py" name="xbox_pub" output="screen">
    </node>
</launch>
